.github/workflows/release.yml

```diff
@@ -778,6 +778,10 @@ jobs:
           npm install -g jsdoc typedoc
       - name: Install USearch from PyPi
         run: pip install usearch
+      - name: Build CJS JS
+        run: |
+          npm ci --ignore-scripts
+          npm run build-js
```

Comment:
Seems like we need this, but not the rest. The rest should probably be reverted 🤔

Reply:
This makes CJS versions that the doxygen seems to expect. Then the other part points to that version.